### ISSUE TITLE
Failure to close low level socket

### ISSUE THREAD

**Issue by poirierlouis** [tiltedphoques/CyberpunkMP#25]
After building and installing with `xmake -y`, `xmake install -o distrib`.
I start the server without problem. A few seconds later I stop it using `CTRL + C`. It shows the following error:
```
src\steamnetworkingsockets\clientlib\steamnetworkingsockets_lowlevel.cpp(3443): Trying to close low level socket support, but we still have sockets open!src\steamnetworkingsockets\clientlib\steamnetworkingsockets_lowlevel.cpp(3381): Assertion Failed: nLastRefCount > 0
```
Is signal handled by CMP? Or is it handled and it fails within Steam SDK only?

**Comment by Zhincore** [comment]
Happens on linux too

**Comment by maximegmd** [comment]
Fixed in [69aa6d5](https://github.com/tiltedphoques/CyberpunkMP/commit/69aa6d557992e05b122f3decbdd925db02d37c25)
